<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RASTR_Widget</class>
 <widget class="QWidget" name="RASTR_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>572</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>Подсистема РАСТР</string>
   </property>
   <widget class="QLabel" name="label_85">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>19</y>
      <width>73</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Режим</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Use">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>66</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>999</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QSpinBox" name="Port">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>45</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>19</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Порт</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>96</y>
      <width>311</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Сервер</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>95</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Определить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Интервал &quot;Я жив&quot;,  сек</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="KeepAliveInterval">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>176</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_86">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Интервал АвтоДК</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_87">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Обогрев камер</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_88">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Наложение титров</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Port2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>296</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>441</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Порт для второго монитора, (&quot;Кедр&quot;)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="AutoDkPeriod">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>176</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="dtInfoToJpg">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>236</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="ThermostatUse">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>236</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
